scripts/convert_cogvideox_to_diffusers.py

```diff
@@ -78,6 +84,7 @@ def replace_up_keys_inplace(key: str, state_dict: Dict[str, Any]):
     "mixins.final_layer.norm_final": "norm_out.norm",
     "mixins.final_layer.linear": "proj_out",
     "mixins.final_layer.adaLN_modulation.1": "norm_out.linear",
+    "mixins.pos_embed.pos_embedding": "patch_embed.pos_embedding",  # Specific to CogVideoX-5b-I2V
```

Comment:
Should we have any if/else to guard that accordingly?

Reply:
This layer is absent in the T2V models actually. It's called `positional_embedding` in T2V which is just sincos PE, while `pos_embedding` here. I think it's safe but going to verify it now

Reply:
Yep, this is safe and should not affect the T2V checkpoints since they follow different layer naming conventions